tests/benchmarks/power_management/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  benchmark.kernel.power_management:
+    filter: CONFIG_SYS_PM_MIN_RESIDENCY_SLEEP_1 and CONFIG_SYS_PM_MIN_RESIDENCY_DEEP_SLEEP_1
```

Comment:
This "filter" filter out all platforms, since no platform enable these config by default. Try these test cases by sanitycheck, project is compiled, but it doesn't run on any platform.

Reply:
The test case by sanitycheck can be run on micro chip board (mec15), you can try with sanitycheck --hardware-map ~/mec1.file --device-testing -T tests/benchmarks/power_management/. This test case depends on these two configurations to run. Currently only mec1501modular_assy6885 and mec15xxevb_assy6853 have both SLEEP and DEEP_SLEEP configurations (See below)
~/zephyrproject/zephyr/boards$ grep -r SYS_PM_MIN_RESIDENCY_SLEEP_1 *
arm/mec1501modular_assy6885/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_SLEEP_1
arm/cc1352r1_launchxl/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_SLEEP_1
arm/mec15xxevb_assy6853/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_SLEEP_1
arm/cc26x2r1_launchxl/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_SLEEP_1

~/zephyrproject/zephyr/boards$ grep -r SYS_PM_MIN_RESIDENCY_DEEP_SLEEP_1 *
arm/mec1501modular_assy6885/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_DEEP_SLEEP_1
arm/mec15xxevb_assy6853/Kconfig.defconfig:config SYS_PM_MIN_RESIDENCY_DEEP_SLEEP_1

Reply:
got it, great!